src/element/sizeHelpers.test.ts

```diff
@@ -24,25 +34,29 @@ describe("getPerfectElementSize", () => {
     expect(width).toBeCloseTo(0, EPSILON_DIGITS);
     expect(height).toBeCloseTo(100, EPSILON_DIGITS);
   });
+
   it("should return adjust height to be width * tan(locked angle)", () => {
     const { height, width } = getPerfectElementSize("arrow", 120, 185);
     expect(width).toBeCloseTo(120, EPSILON_DIGITS);
     expect(height).toBeCloseTo(207.846, EPSILON_DIGITS);
   });
+
   it("should return height equals to width if locked angle is 45 deg", () => {
     const { height, width } = getPerfectElementSize("arrow", 135, 145);
     expect(width).toBeCloseTo(135, EPSILON_DIGITS);
     expect(height).toBeCloseTo(135, EPSILON_DIGITS);
   });
+
   it("should return height:0 and width:0 when width and height are 0", () => {
     const { height, width } = getPerfectElementSize("arrow", 0, 0);
     expect(width).toBeCloseTo(0, EPSILON_DIGITS);
     expect(height).toBeCloseTo(0, EPSILON_DIGITS);
   });
 
-  describe("should respond to SHIFT_LOCKING_ANGLE constant", () => {
-    it("should have only 2 locking angles per section if SHIFT_LOCKING_ANGLE = 45 deg (Math.PI/4)", () => {
-      (constants as any).SHIFT_LOCKING_ANGLE = Math.PI / 4;
+  describe("should respond to SHIFT_LOCKING_ANGLE constant", async () => {
+    it("should have only 2 locking angles per section if SHIFT_LOCKING_ANGLE = 45 deg (Math.PI/4)", async () => {
+      //@ts-ignore
+      constants.SHIFT_LOCKING_ANGLE = Math.PI / 4;
```

Comment:
appears the same, except different way to disable TS. Or any reason for this change?

Reply:
This was needed to mock `constants.ts` file, in jest we can override by just assigning the value but in vite thats not possible hence I need to mock it so it allows me to override the value. I have added a comment regarding the same

Reply:
yep, I meant why change from `(constants as any)`. That was slightly safer :)